scss/tech.scss

```diff
@@ -0,0 +1,40 @@
+.margin-05 {
+  margin: .5rem 0;
+}
+
+.tech-content {
+  margin: 0 auto;
+  padding: 2rem 0;
+  width: 80%;
+}
+
+.tech {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 314px;
+  justify-content: flex-start;
+  padding: 1rem 2.5rem 2.5rem 1rem;
+  position: relative;
+  width: 218px;
+  
+  &-logo {
+    width: 110px;
+  }
+
+  &-description {
+    margin: 1rem 0;
+    text-align: center;
+  }
+    
+  &-border {
+    border: 1px solid $green;
+    border-radius: 5px;
+    height: 100%;
+    position: absolute;
+    top: -11px;
```

Comment:
Always when using absolute positioning, use one vertical and one horizontal parameter. Add right or left position here. Also, maybe a positive bottom will look better instead.